bin.src/validateDrp.py

```diff
@@ -25,32 +25,34 @@
 import os.path
 import sys
 
-from lsst.validate.drp import checkAstrometryPhotometry
+from lsst.validate.drp import validate
 
 
-def defaultData(repo):
-    # List of visits to be considered
-    visits = [849375, 850587]
-
-    # List of CCD to be considered (source calalogs will be concateneted)
-    ccd = [12, 13, 14, 21, 22, 23]
-    filter = 'r'
-
-    # Reference values for the median astrometric scatter and the number of matches
-    good_mag_limit = 21.0
-    medianRef = 25
-    matchRef = 5000
+if __name__ == "__main__":
```

Comment:
Yeah, was almost at the argparse level.  I like argparse and use it for a variety of things outside LSST DM-land.  I held back for two reasons:
1. I don't understand how to combine documentation for the Sphinx doc pages and the nice building of the usage and help message through argparse.  Although maybe already what I did wouldn't register properly for an entry for validateDrp.py in the online docs.
2.  In anticipation of DM-5096 to make this a Task (which will potentially obviate a fraction of this ticket overall, so that's not an entirely consistent position).

If I ever add one more option, I will shift to argparse or Task.